### DISCUSSION TITLE
Integration with Streamlit

### DISCUSSION THREAD

**Discussion by berkorbay** [google/or-tools#2617] [CP-SAT questions] [answered]
**What language and solver does this apply to?**
Python

**Describe the problem you are trying to solve.**
I would like to give my models a "voice" with an easy to use interface like Streamlit.io

**Describe the solution you'd like**
If I could somehow integrate CP Solver callback results (intermediate solutions) to Streamlit components instead of printing to terminal, it would be great. Probably there is an easy way and a minimal example somehow but I could not find it.

**Describe alternatives you've considered**
Currently I am still searching.

**Comment by Mizux** [comment, marked as answer]
related to #2489 IMHO
